Project.toml

```diff
@@ -19,10 +19,10 @@ Zygote = "e88e6eb3-aa80-5325-afca-941959d7151f"
 [compat]
 ColorVectorSpace = "0.9.5"
 Flux = "0.12.6"
+ImageCore = "0.9"
 ImageTransformations = "0.8, 0.9"
-Interpolations = "0.13.3"
 Zygote = "0.6.17"
-ImageCore = "0.9"
+Interpolations = "≥ 0.13.4"
```

Comment:
">= 0.13.4" means "[0.13.0 - 99999999]" no upper bound, which I think is not recommended by the Julia general registry.

See also https://github.com/JuliaRegistries/RetroCap.jl

Reply:
I shall make it only dependent on `0.13.4` for now on. When there are later releases of `Interpolations.jl` I will put a range.